src/main/java/cofh/thermaldynamics/duct/fluid/DuctUnitFluid.java

```diff
@@ -367,13 +367,13 @@ public IFluidHandler getFluidCapability(final EnumFacing from) {
 
 				FluidStack info = grid != null ? grid.myTank.getInfo().fluid : null;
 				int capacity = grid != null ? grid.myTank.getInfo().capacity : 0;
-				return new IFluidTankProperties[] { new FluidTankProperties(info, capacity, isOpen(from), isOpen(from)) };
+				return new IFluidTankProperties[] { new FluidTankProperties(info, capacity, isInput(from.ordinal()), isOutput(from.ordinal())) };
```

Comment:
context: `public FluidTankProperties(@Nullable FluidStack contents, int capacity, boolean canFill, boolean canDrain)`
Sets the canFill and canDrain parameters properly because isOpen() is too generic and always returns false for `ConnectionType.BLOCKED`